docs/tech-spec.md

```diff
@@ -156,35 +156,44 @@ It has been agreed that the scope will NOT cover
 
 ## Deliverables 
 
-1. A PDF / Word document which covers the project lifecycle including; Public Review logs and Final document ready for official publication.
-2. Code in the UN/CEFACT GitHub to deliver the published & maintained JSON-LD vocabulary
+1. A PDF / Word document which covers the project lifecycle including; Public Review logs and the final document ready for official publication.
+2. Code in the UN/CEFACT GitHub (https://github.com/uncefact/vocab) to produce the JSON-LD vocabulary outputs
+3. Delivery of an official and maintained release of the JSON-LD vocabulary published on the UNECE website (https://service.unece.org/trade/uncefact/vocabulary/uncefact.jsonld)
 
 
 ## Maintenance
-Release Cycles and practicalities.
 
+UN/CEFACT release their reference data models twice a year (usually around July and December) the json-ld vocab would be produced as a follow on output from the release of Buy Ship Pay (BSP) reference data model.
+
+1. BSP RDM Release 
```

Comment:
@cmsdroff, some of this needs discussion. I worry this is promising a bit too much.

Reply:
Hi @nissimsan as I understand CEFACT deliverables are twice a year so to be official outputs they would need to be part of this release schedule.

We have an open ticket #87 which I think probably need to be resolved before we include the text above if your not comfortable with it at this stage.  So happy to remove it for time being?

Reply:
@nissimsan changes made for review on next call